coral/src/app/features/approvals/acls/components/AclApprovalsTable.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from "react";
+import {
+  GhostButton,
+  Icon,
+  StatusChip,
+  Table,
+  Flexbox,
+} from "@aivenio/aquarium";
+import { AclRequest } from "src/domain/acl/acl-types";
+import deleteIcon from "@aivenio/aquarium/dist/src/icons/delete";
+import infoSign from "@aivenio/aquarium/dist/src/icons/infoSign";
+import loadingIcon from "@aivenio/aquarium/dist/src/icons/loading";
+import tickCircle from "@aivenio/aquarium/dist/src/icons/tickCircle";
+import {
+  requestStatusChipStatusMap,
+  requestStatusNameMap,
+} from "src/app/features/approvals/utils/request-status-helper";
+
+type Props = {
+  aclRequests: AclRequest[];
+  activePage: number;
+  totalPages: number;
+  onDetails: (aclRequest: AclRequest) => void;
+  onApprove: (aclRequest: AclRequest) => void;
+  onDecline: (AclRequest: AclRequest) => void;
+};
+
+export default function AclApprovalsTable({
+  aclRequests,
+  activePage,
+  totalPages,
+  onDetails,
+  onApprove,
+  onDecline,
+}: Props) {
+  const [isLoading, setLoading] = useState<boolean>(false);
+  return (
+    <Table ariaLabel={`Acl requests, page ${activePage} of ${totalPages}`}>
```

Comment:
I see you have chosen to use the `Table` component from `aquarium`. I believe it will eventually be deprecated in favor of `DataTable`, which is why we elected to use `DataTable` for the Approval pages. I think it would be nice to keep it consistent ^^